app/views/constituencies/current.html.haml

```diff
@@ -1,3 +1,5 @@
+- title t('.title', current: t('current'), constituencies: t('constituencies_title'))
```

Comment:
if we did something like - `page_title = title( t('.title', current: t('current'), constituencies: t('constituencies_title')) )`

Then you could do `%h1= page_title` to remove duplication